pkg/reconciler/brokercell/brokercell.go

```diff
@@ -220,7 +221,7 @@ func (r *Reconciler) shouldGC(ctx context.Context, bc *intv1alpha1.BrokerCell) b
 }
 
 func (r *Reconciler) delete(ctx context.Context, bc *intv1alpha1.BrokerCell) pkgreconciler.Event {
-	if err := r.RunClientSet.InternalV1alpha1().BrokerCells(bc.Namespace).Delete(bc.Name, nil); err != nil {
+	if err := r.RunClientSet.InternalV1alpha1().BrokerCells(bc.Namespace).Delete(ctx, bc.Name, metav1.DeleteOptions{}); err != nil {
```

Comment:
do deleteOptions need to be an empty object, or can it be nil as before?
Same thing for all other places..

Reply:
Yes, otherwise I'm getting `cannot convert nil (untyped nil value) to "k8s.io/apimachinery/pkg/apis/meta/v1".DeleteOptions`

Reply:
ack